vignettes/development_process.Rmd

```diff
@@ -0,0 +1,39 @@
+3. Familiarise yourself with the admiral [programming strategy](https://roche-gsk.github.io/admiral/articles/programming_strategy.html), and then make the required code updates
+4. Before making a pull request , check the [Pull Request Review Guidance](https://roche-gsk.github.io/admiral/articles/pr_review_guidance.html) & the following checklist of common things developers miss:
+  a. Is all your code formatted according to the [tidyverse](https://style.tidyverse.org/) style guide? 
+  b. Did you create/add appropriate [unit tests](https://roche-gsk.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)?
+  c. If you removed/replaced any function did you fully follow the [deprecation guidance](https://roche-gsk.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
+  d. Did you update the [documentation](https://roche-gsk.github.io/admiral/articles/programming_strategy.html#function-header-documentation-)? If so, remember to run `devtools::document()` and include the updated DESCRIPTION, NAMESPACE, and files in man/  
+  e. Does your code update have any impact on the [ADaM template](https://roche-gsk.github.io/admiral/articles/admiral.html#starting-a-script-1) R scripts stored in inst/templates?
+  f. Does your code update have any impact on the vignettes stored in vignettes?
+  g. Did you update the Changelog (NEWS.md)?
+5. Once happy with all the updates, make a [pull request](https://roche-gsk.github.io/admiral/articles/git_usage.html#working-with-feature-branches-1) to merge to the development branch (devel) and link the issue so that it closes after successful merging
+6. Check that there are no merge conflicts. If there are any, fix them before requesting review. See [solving merge conflicts](https://roche-gsk.github.io/admiral/articles/git_usage.html#solving-merge-conflicts) guidance.
+7. Check the results of the automated R-CMD and lintr checks and if any issues consult this [guide](https://roche-gsk.github.io/admiral/articles/pr_review_guidance.html#common-r-cmd-check-issues-1)
+8. Assign a reviewer from the admiral core development team – this could be anyone you discussed the issue with previously via Slack or GitHub. If unsure, add a comment that the pull request is ready for review and add the @Roche-GSK/admiral-devs tag to it
+9. Once the review is completed, the reviewer will merge the PR and click to “delete branch”
+
+_Finally, just a note to say from the core developers that we hugely appreciate you taking the time to contribute to admiral. Don’t be offended if during review we send requests back to you, as the expectations are high so that we can ensure the admiral codebase remains robust and consistent. The best way to learn here is always to jump in and get involved, so please don’t be afraid you’ll make mistakes along the way – we all have and continue to do so, and that’s what the reviews are for. **Welcome to our admiral community!**_
```

Comment:
Maybe add something saying that whenever they get stuck with any of the steps outlined above they should feel free to contact a dev team member for support.